python/py-pytest-html/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pytest-html
+name                py-${real_name}
+version             2.0.1
+revision            0
+categories-append   devel
+
+license             Mozilla-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Plugin for generating HTML reports for pytest results
+
+long_description    pytest-html is a plugin for pytest that generates\
+                    a HTML report for the test results.
+
+homepage            https://pypi.python.org/pypi/${real_name}
+
+master_sites        pypi:p/${real_name}/
+distname            ${real_name}-${version}
+
```

Comment:
in the three lines above, use ```python.rootname``` instead of ```real_name```

Reply:
Thanks done.